db/migrate/20230401123456_create_items.rb

```diff
@@ -0,0 +1,12 @@
+class CreateItems < ActiveRecord::Migration[7.0]
+  def change
+    create_table :items do |t|
+      t.string :name, null: false
+      t.decimal :price, precision: 10, scale: 2, null: false
+      t.decimal :discounted_price, precision: 10, scale: 2, null: false
+      t.references :collection, null: false, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
The Items table migration sets both 'price' and 'discounted_price' as non-nullable with a default of 0. This could lead to items appearing as free by default. Consider if a default value is appropriate or if it should be left to the application logic to enforce the presence of a price.